visitors/http_rest/spec_visitor_context.go

```diff
@@ -25,6 +25,13 @@ func (t HttpValueType) String() string {
 type SpecVisitorContext interface {
 	visitors.Context
 
+	// Identifies the REST field being visited when combined with IsArg,
+	// IsResponse, and GetValueType. For fields outside of bodies and
+	// authorization headers, the first component of the path is a FieldName
+	// identifying the path argument, query variable, header, or cookie in which
+	// the field is located.
+	GetFieldLocation() []FieldLocationElement
```

Comment:
`GetRelativeFieldLocation` to avoid confusion with the other definition of `FieldLocation`?

Reply:
Commit 940575f calls these "paths" instead of "locations".